code/config/lib/Loot/Config/CLI.hs

```diff
@@ -0,0 +1,89 @@
+{- This Source Code Form is subject to the terms of the Mozilla Public
+ - License, v. 2.0. If a copy of the MPL was not distributed with this
+ - file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ -}
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+{-# LANGUAGE DataKinds        #-}
+{-# LANGUAGE KindSignatures   #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE TypeOperators    #-}
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
Why can't this be handled globally (see https://github.com/int-index/foundry/blob/master/foundry.cabal#L93)?

Reply:
OK!